packages/node_modules/@node-red/nodes/locales/en-US/function/90-exec.html

```diff
@@ -80,4 +80,5 @@ <h4>Killing processes</h4>
     <p>If the node has more than one process running then <code>msg.pid</code> must also be set with the value of the PID to be killed.</p>
     <p>If a value is provided in the <code>Timeout</code> field then, if the process has not completed when the specified number of seconds has elapsed, the process will be killed automatically</p>
     <p>Tip: if running a Python app you may need to use the <code>-u</code> parameter to stop the output being buffered.</p>
+    <p>The <code>Hide windows</code> option can be set to hide shell windows under Windows.</p>
```

Comment:
should we emphasis that this ONLY has an effect under Windows ? or does it do something under Linux/Mac also ?